internal/daemon/controller/auth/auth.go

```diff
@@ -694,3 +695,123 @@ func GetTokenFromRequest(ctx context.Context, kmsCache *kms.Kms, req *http.Reque
 
 	return publicId, encryptedToken, uint32(receivedTokenType)
 }
+
+// ScopesAuthorizedForList retrieves and returns all scopes where a user is authorized
+// to perform a *list* action on. It looks recursively from `rootScopeId`.
+func (r *VerifyResults) ScopesAuthorizedForList(ctx context.Context, rootScopeId string, resourceType resource.Type) (map[string]*scopes.ScopeInfo, error) {
+	const op = "scopeids.AuthorizedForList"
```

Comment:
```suggestion
	const op = "auth.(VerifyResults).ScopesAuthorizedForList"
```

Reply:
There's always something I miss 😆, thanks for spotting, addressed in [8d45a6a](https://github.com/hashicorp/boundary/pull/2336/commits/8d45a6a21cf97b37362066fadcfbc8380019df3c)